articles/tech-resources/enabling-multiple-custom-tracking-domains.md

```diff
@@ -47,6 +47,8 @@ To configure one or more custom tracking domains through the SparkPost user inte
     ![](media/enabling-multiple-custom-tracking-domains/xU06kJ2lac6YGEIi8KlReq8OaF1kfX.png)
 1. Complete the Sending Domain Name for each sending domain and select the Linked Tracking Domain you want to associate with it.
 1. Click the Add Domain or _Update Tracking Domain_ button to complete the association.
+Please note in order to associate a tracking domain with a sending doamin you need to make you have verified the trakcing domain first. 
```

Comment:
I don't want this note to get lost.  Can we move it before the first step?

## Associate your Tracking domain(s) with one or more Sending domains

To associate custom tracking domains to sending domains through the SparkPost user interface, the custom tracking domains must be verified first and then follow the steps below.

1. Select Account and then Sending Domains from the main menu.